system-parachains/people/people-kusama/src/lib.rs

```diff
@@ -134,8 +136,8 @@ pub type SignedBlock = generic::SignedBlock<Block>;
 /// BlockId type as expected by this runtime.
 pub type BlockId = generic::BlockId<Block>;
 
-/// The SignedExtension to the basic transaction logic.
-pub type SignedExtra = (
+/// The TransactionExtension to the basic transaction logic.
+pub type TxExtension = (
```

Comment:
(we can also add `WeightReclaim` in this pipeline)

Reply:
I noticed `WeightReclaim` was introduced in `frame-system@40.1.0` (`stable2503`). So, I'll be skipping this for now, because we're a bit time-strechted to release this upgrade.